src/connectedk8s/azext_connectedk8s/tests/latest/data/cli-test-aks-qkwd4jje7tm-config.yaml

```diff
@@ -0,0 +1,20 @@
+    token: 6f92f583edd15914411a50fe51117e72269670f854086fc8bd2efa9aa60104fbcfea10484a21df4fd0cb1936cf33ccb0af2c80426c6119d6e7f5c5454c686ab7
```

Comment:
Is it OK to expose these data?

Reply:
Since we delete the aks cluster at the end, it should not be a problem i guess